owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/settings/SettingsSecurityViewModelTest.kt

```diff
@@ -200,4 +204,18 @@ class SettingsSecurityViewModelTest : ViewModelTest() {
         }
     }
 
+    @Test
+    fun `set pref is security enforced enabled - ok - true`() {
+        every { securityViewModel.isSecurityEnforcedEnabled() } returns true
+
+        assertTrue(contextProvider.getBoolean(R.bool.passcode_enforced))
+    }
+
+    @Test
+    fun `set pref is security enforced enabled - ok - false`() {
+        every { securityViewModel.isSecurityEnforcedEnabled() } returns false
+
+        assertFalse(contextProvider.getBoolean(R.bool.passcode_enforced))
+    }
+
```

Comment:
Hm 🤔 , I don't think these unit tests are valid. Here, you are not actually testing the method `isSecurityEnforcedEnabled()`. You don't have to mock it for the tests to work, it is not like in UI tests. You should mock the necessary stuff of the content of the method, and check whether what the method returns is what you expect. Therefore, you mustn't mock the method itself, otherwise there's no testing and no coverage here! Tip: think or check the coverage the method provides 👍 
Also, would be cool if you added a `verify` clause to check if the method executes the lines you expect.
You can find an example of all of this in the `is help enabled - ok - true()` method in `SettingsMoreViewModelTest`.

Reply:
The tests are new to me, sorry for the mistakes, it's a bit complicated at the beginning. thanks for teaching me!

Reply:
You don't have to be sorry 😄